CHANGELOG.md

```diff
@@ -1,6 +1,4 @@
-# Changelog
-
-## Next version
+## 2021-11-05 v0.6
```

Comment:
I wonder if we should add a brief description of _benefits_ for users? I.e. we mention "store configurable" but we don't say what this fixes. Ditto re websockets paving way for native browser usage. Could be a paragraph at top of release notes that focuses on benefits to end users. WDYT?

Also I think fine to include RLN changes, prefacing with that it is experimental and under compiler flag.

Otherwise LGTM!

Reply:
Good idea. RLN changes and top-level summary of benefits added in 6bc46d6